src/Backends/ColorSettings.vala

```diff
@@ -0,0 +1,90 @@
+/*
+* Copyright 2020 elementary, Inc. (https://elementary.io)
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 3 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*
+*/
+
+public class SettingsDaemon.Backends.ColorSettings : GLib.Object {
+    public unowned PantheonShell.Pantheon.AccountsService accounts_service { get; construct; }
+
+    private GLib.Settings color_settings;
+
+    public ColorSettings (PantheonShell.Pantheon.AccountsService accounts_service) {
+        Object (accounts_service: accounts_service);
+    }
+
+    construct {
+        color_settings = new GLib.Settings ("io.elementary.settings-daemon.plugins.color");
```

Comment:
I'm not sure we need to namespace this `plugins` since (unlike gsd) we don't have a plugin architecture

Reply:
I just copied it from the other project. @danrabbit Do you have a preference as to what it should be called?

Maybe `io.elementary.settings-daemon.prefers-color-scheme`?

Reply:
Yeah that seems better to me. Thanks!